includes/rest-api/class-sensei-rest-api-v1.php

```diff
@@ -19,6 +20,8 @@ class Sensei_REST_API_V1 {
 
 	/**
 	 * Sensei_REST_API constructor.
+	 *
+	 * @deprecated 3.11.0
 	 */
 	public function __construct() {
 		add_action( 'rest_api_init', array( $this, 'register' ) );
```

Comment:
This constructors doesn't have the `_deprecated_function`

Reply:
Yeah this was intentional as we still create the objects and it they would create warnings in the logs.